sql/moz-fx-cjms-nonprod-9a36/cjms_bigquery/subscriptions_v1/query.sql

```diff
@@ -38,10 +38,10 @@ attributed_flows AS (
     subscription_created
 ),
 attributed_subs AS (
-  -- last subscription for each flow, for 1:1 relationship between flow and subscription
+  -- first subscription for each flow, for 1:1 relationship between flow and subscription
   SELECT
     flow_id,
-    ARRAY_AGG(subscription_id ORDER BY subscription_created DESC LIMIT 1)[
+    ARRAY_AGG(subscription_id ORDER BY subscription_created LIMIT 1)[
```

Comment:
per https://github.com/mozilla/bigquery-etl/pull/2823#discussion_r835615236 I discussed this with product, and we do indeed want first subscription by flow.